x-pack/plugins/observability/public/components/custom_threshold/components/rule_condition_chart/rule_condition_chart.tsx

```diff
@@ -47,13 +58,15 @@ const getOperationTypeFromRuleAggType = (aggType: AggType): OperationType => {
 export const getBufferThreshold = (threshold?: number): string =>
   (Math.ceil((threshold || 0) * 1.1 * 100) / 100).toFixed(2).toString();
 
-export function PreviewChart({
```

Comment:
The `PreviewChart` works well in context of rule configuration, when the purpose is to show user preview of data flow so that can define appropriate threshold. In context of alert details page, we are showing data at the time alert triggered and some data before and after alert to analyse. I think `RuleConditionChart` is a common name that would work for both purposes. We can rename it to something else to cover both use cases, so if you have any other suggestions, let me know.

Reply:
I see your point, no I don't have a better suggestion.

Would you mind renaming this component to PreviewChart in [here](https://github.com/elastic/kibana/pull/175513/files#diff-1057e1ee01ae9e827ccaa2075d9deadb0e40b534d16d6434abd97a1132534f84R45) to make it easier to find in the code?

```
import { RuleConditionChart as PreviewChart } from './components/rule_condition_chart/rule_condition_chart';
```